SPECS/prism/prism.spec

```diff
@@ -0,0 +1,63 @@
+%global debug_package %{nil}
+%define our_gopath %{_topdir}/.gopath
+
+Summary:        Azure Linux image customization tool
+Name:           prism
+Version:        0.12.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://github.com/microsoft/azure-linux-image-tools
+Source0:        https://github.com/microsoft/azure-linux-image-tools/archive/refs/tags/v%{version}.tar.gz#/azure-linux-image-tools-%{version}.tar.gz
+BuildArch: x86_64
+BuildRequires: golang >= 1.21
+Requires: qemu-img
+Requires: rpm
+Requires: coreutils
+Requires: util-linux
+Requires: /usr/bin/udevadm
+Requires: openssl
+Requires: sed
+Requires: createrepo_c
+Requires: squashfs-tools
+Requires: cdrkit
+Requires: parted
+Requires: e2fsprogs
+Requires: dosfstools
+Requires: xfsprogs
+Requires: zstd
+Requires: veritysetup
+Requires: grub2
+Requires: grub2-pc
+Requires: systemd-ukify
```

Comment:
systemd-ukify is run on host so I don't think it is needed as a dependency. Lanze can confirm.